src/fragments/lib/auth/ios/signin_next_steps/80_confirm_totp_mfa.mdx

```diff
@@ -0,0 +1,57 @@
+If the next step is `confirmSignInWithTOTPCode`, you should prompt the user to enter the TOTP code from their associated authenticator app during set up. The code is a six-digit number that changes every 30 seconds. The user must enter the code before the 30-second window expires.
```

Comment:
Technically, there's a +1/-1 window Cognito uses to validate codes.

Reply:
Do we need to call that out?